library/include/hipblas.h

```diff
@@ -572,6 +572,12 @@ typedef enum
     HIPBLAS_COMPUTE_64F_PEDANTIC = 8, /**< compute will be exactly 64-bit precision */
     HIPBLAS_COMPUTE_32I          = 9, /**< compute will be at least 32-bit integer precision */
     HIPBLAS_COMPUTE_32I_PEDANTIC = 10, /**< compute will be exactly 32-bit integer precision */
+    HIPBLAS_COMPUTE_32F_FAST_8F  = 100, /**< 32-bit input can use fp8 compute */
```

Comment:
HIPBLAS_COMPUTE_32F_FAST_8F_FNUZ
Refer to https://github.com/ROCm/HIP/blob/9eb37e9947ff630d623e0172dc4150a6d640280f/include/hip/library_types.h#L62